fetch.bs

```diff
@@ -5434,11 +5434,9 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
    <a for="fetch timing info">post-redirect start time</a>, and <var>fetchParams</var>'s
    <a for="fetch params">cross-origin isolated capability</a>.
 
-   <li><p>If <var>connection</var> is not an HTTP/2 connection, <var>request</var>'s
-   <a for=request>body</a> is non-null, and <var>request</var>'s <a for=request>body</a>'s
-   <a for=body>source</a> is null, then <a for="header list">append</a>
-   (`<code>Transfer-Encoding</code>`, `<code>chunked</code>`) to <var>request</var>'s
-   <a for=request>header list</a>.
+   <li><p>If <var>connection</var> is neither an HTTP/2 nor an HTTP/3 connection,
+   <var>request</var>'s <a for=request>body</a> is non-null, and <var>request</var>'s
+   <a for=request>body</a>'s <a for=body>source</a> is null, then return a <a>network error</a>.
```

Comment:
Can we talk about it not being an HTTP/1 connection? Presumably HTTP/2+ has this covered, not just 2 and 3.

Reply:
Done.